docs/rfcs/0002/readme.md

```diff
@@ -178,8 +178,8 @@ Peter submits the job to the WLM. WLM guides the job through the workflow states
             metadata:
                 name: my-job-container-my-foo
             data:
-                DW_JOB_foo-local-storage:             type=gfs2   mount-type=indexed-mount
-                DW_PERSISTENT_foo-persistent-storage: type=lustre mount-type=mount-point
+                DW_JOB_foo_local_storage:             mount-type=indexed-mount
```

Comment:
- Can "property-like" keys not take hyphens?
- LLNL may want a specific file format. For example, if they already have a package for reading JSON, we may want to default the format to JSON. Before you start coding I would get confirmation on the desired behavior.